### DISCUSSION TITLE
Weird error when executing queryRaw

### DISCUSSION THREAD

**Discussion by MaxiSanchez600** [prisma/prisma#25012] [Q&A]
### Bug description

Hi everyone, this is Maxi from Argentina.

Im having this weird issue when executing some queryraws with prisma client. This is the query:

```js
client.$queryRaw`SELECT *,
       ("createdAt" AT TIME ZONE 'UTC' AT TIME ZONE ${timeZone}) AS "createdAt"
                      FROM (
                          SELECT *, ROW_NUMBER() OVER(PARTITION BY DATE_TRUNC('hour', "createdAt") ORDER BY "createdAt" DESC) as row_num FROM "WalletSnapshots" ws
                          WHERE ("createdAt" AT TIME ZONE 'UTC' AT TIME ZONE ${timeZone}) >= CAST(${fromDate} AS TIMESTAMP)
                            AND ("createdAt" AT TIME ZONE 'UTC' AT TIME ZONE ${timeZone}) <=  CAST(${toDate} AS TIMESTAMP)
                            AND "walletId" = ${walletId}
                      ) AS subquery
                      WHERE row_num = 1`;
```
                      
The thing here is: This is working fine when testing the code locally. However, when I try to build and run my API in a docker image, this query starts to fail with the following message:

```
TypeError: e2.map is not a function
    at wl (/app/packages/api/dist/chunk-FM4H3MFA.js:890183:17)
    at async _class33.getLastBalancesFromPeriod (/app/packages/api/dist/chunk-FM4H3MFA.js:943123:29)
    at async /app/packages/api/dist/chunk-FM4H3MFA.js:941751:24
    at async Object.getLastBalancesFromPeriod (/app/packages/api/dist/chunk-FM4H3MFA.js:944538:26)
```

I think this might have something to do with the raw query mapping, but I just dont understand it.

Thanks,
Maxi

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by Weakky** [comment]
Hey @MaxiSanchez600,

Your error does not seem to come from Prisma, but from your own business logic. Would you mind sharing the content of the  `getLastBalancesFromPeriod ` function so that we can help you further? Thanks

**Reply by ludralph** [reply]
Hi @MaxiSanchez600 👋 

It would be really helpful if you could provide the additional information that we had asked about last time. If your problem is resolved by now or somehow doesn't feel relevant any more, you can simply close the Discussion as well.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
